apps/e2e/cypress/support/utils/createProjectInformation.js

```diff
@@ -121,6 +122,7 @@ export const projectInformation = (options = {}) => {
 		internalDateFullFormatted,
 		internalDateMonth,
 		internalDateYear,
+		isMaterialChange,
```

Comment:
We should add tests to validate the `Material change` field exists  and to validate the ability to change the value when when `isMaterialchange` is true -  they can run when the FF is ON

Reply:
@francescobrenci-AR I left this variable in the codebase by mistake. This e2e journey involves the creation of the case. It looks like currently there are no e2e tests for the edit fields and pages.